filemanager/filemanager.go

```diff
@@ -68,6 +68,8 @@ type Settings struct {
 	Config   map[string]interface{}
 	Logger   logger.Logger
 	Conf     *config.Config
+
+	DoNotOverWriteGCS bool
```

Comment:
can you add a comment on what this field will do when set to true?
```suggestion
	DoNotOverwriteFileForGCS bool
```

Reply:
done

Reply:
Wdyt for naming it as `DisableGCSOverwrite`?

Reply:
thinking back on it `overwrite` can lead to some confusion.
a simple upload if not exists type setting might make more sense.
`GCSUploadIfNotExists` wdyt?